en/about/faq.md

```diff
@@ -0,0 +1,43 @@
+# Frequently Asked Questions (FAQ)
+
+## Users
+
+<dl>
+  <dt>How efficient is MAVLink?</dt>
+  <dd>MAVLink is a very efficient protocol. It has just 8 bytes overhead per packet, including start sign and packet drop detection.</dd>
```

Comment:
This is accurate for MAVLink 1:
![image](https://user-images.githubusercontent.com/5368500/42303471-d58fb58e-8064-11e8-9fee-87c6e12c0e30.png)
But what about for MAVLink 2. From docs it looks like 
- the message ID grows from 1 byte to 4bytes
- There is an optional added signature.

1. How many bytes are the signature?
1. Is the signature added after the CRC? If not, where is it?
1. Are there any other new bytes/things?

I think for MAVLink 2 i looks like this (see questions).
![image](https://user-images.githubusercontent.com/5368500/42303773-2016e482-8066-11e8-9c46-8f31f047400c.png)

1. How does the variable size array stuff affect this? 
  - My assumption is "not at all" - the length and CRC are calculated after the payload has had the empty end of the largest (final) array truncated.
   - Correct?
, but what is the overhead for MAVLink2?
I guess that the packet anatomy looks like this?

Reply:
Ah, this is documented, but in "wrong place" - [Protocol overview](https://mavlink.io/en/). I'll check that out then update above.